### PR TITLE
clarify how to go to EAP menu

### PR DESCRIPTION
**Pull request by MaratMussabekov**
#5925

### CODE REVIEW

**Comment by mypil** [comment]
@JohanFreelancer9 - kindly do the copy editor review for this PR. Thanks.

File: windows/client-management/advanced-troubleshooting-802-authentication.md [hunk 1]
```diff
@@ -59,7 +59,7 @@ First, validate the type of EAP method being used:
  
 ![eap authentication type comparison](images/comparisontable.png)
 
-If a certificate is used for its authentication method, check if the certificate is valid. For server (NPS) side, you can confirm what certificate is being used from the EAP property menu:
+If a certificate is used for its authentication method, check if the certificate is valid. For server (NPS) side, you can confirm what certificate is being used from the EAP property menu (In **NPS snapp-in** go to **Policies** -> **Network Policies** -> Right click on the policy and select **Properties** -> In the appeared window go to **Constraints** tab and select **Authentication methods** section):
```
Comment (JohanFreelancer9) [inline windows/client-management/advanced-troubleshooting-802-authentication.md]:
```suggestion
If a certificate is used for its authentication method, check if the certificate is valid. For server (NPS) side, you can confirm what certificate is being used from the EAP property menu. In **NPS snapp-in**, go to **Policies** > **Network Policies**. Right click on the policy and select **Properties**. In the pop-up window, go to the **Constraints** tab and select select the **Authentication methods** section.
```

**Comment by mypil** [comment]
@Dansimp - PR has been copy edited and is ready for final review. This is based on issue #5925

Thank you.

cc: @kenwith

File: windows/client-management/advanced-troubleshooting-802-authentication.md [hunk 2]
```diff
@@ -59,7 +59,7 @@ First, validate the type of EAP method being used:
  
 ![eap authentication type comparison](images/comparisontable.png)
 
-If a certificate is used for its authentication method, check if the certificate is valid. For server (NPS) side, you can confirm what certificate is being used from the EAP property menu:
+If a certificate is used for its authentication method, check if the certificate is valid. For server (NPS) side, you can confirm what certificate is being used from the EAP property menu. In **NPS snapp-in**, go to **Policies** > **Network Policies**. Right click on the policy and select **Properties**. In the pop-up window, go to the **Constraints** tab and select select the **Authentication methods** section.
```
Comment (JohanFreelancer9) [inline windows/client-management/advanced-troubleshooting-802-authentication.md]:
```suggestion
If a certificate is used for its authentication method, check if the certificate is valid. For server (NPS) side, you can confirm what certificate is being used from the EAP property menu. In **NPS snapp-in**, go to **Policies** > **Network Policies**. Right click on the policy and select **Properties**. In the pop-up window, go to the **Constraints** tab and select the **Authentication methods** section.
```

**Comment by illfated** [comment]
@JohanFreelancer9 : Just a little bit funny, you added a copy-paste duplication error to the text in your previous suggestion. 😄

File: windows/client-management/advanced-troubleshooting-802-authentication.md (hunk 2 above)
Comment (mypil) [reply windows/client-management/advanced-troubleshooting-802-authentication.md]: @MaratMussabekov - please merge this additional commit. Thank you.

**Comment by mypil** [comment]
@Dansimp - PR has been copy edited and is ready for final review. This is based on issue #5925

Thank you.

cc: @kenwith

File: windows/client-management/advanced-troubleshooting-802-authentication.md [hunk 3]
```diff
@@ -59,7 +59,7 @@ First, validate the type of EAP method being used:
  
 ![eap authentication type comparison](images/comparisontable.png)
 
-If a certificate is used for its authentication method, check if the certificate is valid. For server (NPS) side, you can confirm what certificate is being used from the EAP property menu:
+If a certificate is used for its authentication method, check if the certificate is valid. For server (NPS) side, you can confirm what certificate is being used from the EAP property menu. In **NPS snapp-in**, go to **Policies** > **Network Policies**. Right click on the policy and select **Properties**. In the pop-up window, go to the **Constraints** tab and select the **Authentication methods** section.
```
Comment (illfated) [inline windows/client-management/advanced-troubleshooting-802-authentication.md]: Correction of minor typo ("snapp-in" `->` snap-in):
```suggestion
If a certificate is used for its authentication method, check if the certificate is valid. For server (NPS) side, you can confirm what certificate is being used from the EAP property menu. In **NPS snap-in**, go to **Policies** > **Network Policies**. Right click on the policy and select **Properties**. In the pop-up window, go to the **Constraints** tab and select the **Authentication methods** section.
```

**Comment by illfated** [comment]
Sorry for the delayed correction. I did not notice it before the last update.

File: windows/client-management/advanced-troubleshooting-802-authentication.md (hunk 3 above)
Comment (mypil) [reply windows/client-management/advanced-troubleshooting-802-authentication.md]: @MaratMussabekov - please merge this additional commit. Thank you.

**Comment by illfated** [comment]
@JohanFreelancer9 : May I ask you to add your approval in the [Files changed] page, if you find the current content ready for final review & merge?

File: windows/client-management/advanced-troubleshooting-802-authentication.md [hunk 4]
```diff
@@ -59,7 +59,7 @@ First, validate the type of EAP method being used:
  
 ![eap authentication type comparison](images/comparisontable.png)
 
-If a certificate is used for its authentication method, check if the certificate is valid. For server (NPS) side, you can confirm what certificate is being used from the EAP property menu:
+If a certificate is used for its authentication method, check if the certificate is valid. For server (NPS) side, you can confirm what certificate is being used from the EAP property menu. In **NPS snap-in**, go to **Policies** > **Network Policies**. Right click on the policy and select **Properties**. In the pop-up window, go to the **Constraints** tab and select the **Authentication methods** section.
```
Comment (JohanFreelancer9) [inline windows/client-management/advanced-troubleshooting-802-authentication.md]:
```suggestion
If a certificate is used for its authentication method, check if the certificate is valid. For server (NPS) side, you can confirm what certificate is being used from the EAP property menu. In **NPS snap-in**, go to **Policies** > **Network Policies**. Right click on the policy and select **Properties**. In the pop-up window, go to the **Constraints** tab and select the **Authentication Methods** section.
```

Comment (mypil) [reply windows/client-management/advanced-troubleshooting-802-authentication.md]: @MaratMussabekov - please merge this additional commit. Thank you.

**Pull request merged**